src/main/kotlin/racingcar/Force.kt

```diff
@@ -0,0 +1,11 @@
+package racingcar
+
+data class Force(val value: Int) {
+    operator fun compareTo(other: Force): Int {
+        return this.value - other.value
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return this.value.compareTo(other.value)
```